src/Resources/skeleton/verifyEmail/VerifyEmail.tpl.php

```diff
@@ -0,0 +1,54 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bridge\Twig\Mime\TemplatedEmail;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+use SymfonyCasts\Bundle\VerifyEmail\Exception\VerifyEmailExceptionInterface;
+use SymfonyCasts\Bundle\VerifyEmail\VerifyEmailHelperInterface;
+
+class <?= $class_name; ?><?= "\n" ?>
+{
+    private $helper;
+    private $mailer;
+    private $manager;
+
+    public function __construct(VerifyEmailHelperInterface $helper, MailerInterface $mailer, EntityManagerInterface $manager)
+    {
+        $this->helper = $helper;
+        $this->mailer = $mailer;
+        $this->manager = $manager;
+    }
+
+    public function sendEmailConfirmation(string $validationRouteName, UserInterface $user, TemplatedEmail $email): void
```

Comment:
```suggestion
    public function sendEmailConfirmation(string $verifyEmailRouteName, UserInterface $user, TemplatedEmail $email): void
```

I also keep debating back and forth whether we should have the route argument at all. I mean, when would they need to change it? Even if they built some system where you could verify multiple emails, they would probably just update the code inside that *one* controller, right?

Reply:
I thought about this early on too, ended up telling myself - If it were me, and I changed the route, it sure would be "convenient" to only have to set this in the controller vs hunting down a `$router->generate` type of method.. 

I'll come back to this after I squash some more squawks.. leaning towards simplicity for the this class, but open to everything..